### PR TITLE
feat: lazy load images and iframes

### PR DESCRIPTION
**Pull request by aslafy-z**
See https://github.com/mfranzke/loading-attribute-polyfill

### CODE REVIEW

**Comment by aslafy-z** [comment]
initial: https://lighthouse-dot-webdotdevsite.appspot.com//lh/html?url=http%3A%2F%2Fdevswag.io

with this PR: https://lighthouse-dot-webdotdevsite.appspot.com//lh/html?url=https%3A%2F%2F5eae43861a9d15000750fb8c--peaceful-chandrasekhar-efde8e.netlify.app%2F

**Comment by aslafy-z** [comment]
@vikaspotluri123 looks good to you?

File: src/pug/includes/body.pug [hunk 1]
```diff
@@ -63,9 +63,10 @@
 					div.flex.img-container
 						a(href=swag.reference target="_blank")
 							picture
-								source(type="image/webp" srcset=swag.images.webp)
-								source(type="image/jpeg" srcset=swag.images.jpeg)
-								img(src=swag.images.jpeg alt=swag.name + " swag you can get")
+								no-script.loading-lazy
```
Comment (vikaspotluri123) [inline src/pug/includes/body.pug:66]: Are these supposed to have `-`?

Comment (aslafy-z) [reply src/pug/includes/body.pug]: Noop, you're right! Fixed it.

**Pull request merged**